### PR TITLE
Add tests for try.sh and recommend to install java 11

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: easy/test_is_up.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python2
```
Comment (smakalias) [inline easy/test_is_up.py]: out of curiosity: why `python2`?

File: easy/test_is_up.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import time
+from urllib import urlopen
+
+
+def wait_for(predicate, timeout=300):
+    amount_slept = 0
+    while not predicate():
+        time.sleep(0.5)
```
Comment (smakalias) [inline easy/test_is_up.py]: nitpicking: use a var?

Comment (smakalias) [inline easy/test_is_up.py]: nitpicking: use var?

File: easy/test_is_up.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        amount_slept += 0.5
+        if amount_slept >= timeout:
+            raise SystemExit('Timeout waiting for ' + predicate.__name__)
```
Comment (smakalias) [inline easy/test_is_up.py]: I remember I used recently a Timeout exception, would that be more appropriate in this context?

File: easy/test_is_up.py (hunk 1 above)
Comment (mfussenegger) [reply easy/test_is_up.py]: Because this script is run inside old distributions which ship with python 2

File: easy/test_is_up.py (hunk 3 above)
Comment (mfussenegger) [reply easy/test_is_up.py]: I think it doesn't exist in python2

**Pull request merged**